_build/builder/BuildDocs.py

```diff
@@ -73,27 +88,19 @@ def Setup(self):
 
     def DoTask(self):
         tmpDir = join(self.buildSetup.tmpDir, "chm")
-        Prepare()
-        #warnings.simplefilter('ignore', DeprecationWarning)
-        sphinx.build_main([
-            None,
-            # '-q',  # be quiet
-            "-a",  # always write all output files
-            "-b", "htmlhelp",
-            "-E",  # Don’t use a saved environment (the structure
-                   # caching all cross-references),
-            "-N",  # Prevent colored output.
-            "-P",  # (Useful for debugging only.) Run the Python debugger,
-                   # pdb, if an unhandled exception occurs while building.
-            "-D", "release=%s" % self.buildSetup.appVersion,
-            "-D", "templates_path=[]",
-            "-d", EncodePath(join(self.buildSetup.tmpDir, ".doctree")),
-            '-v', '-v',  # verbosity, can be given up to three times
-            # write warnings and errors to file
-            # '-w', join('output', 'sphinx_log.txt'),
-            EncodePath(DOCS_SOURCE_DIR),
-            tmpDir,
-        ])
+        WritePluginList(join(self.buildSetup.docsDir, "pluginlist.rst"))
```

Comment:
I am thinking we should not include a list of plugins in the docs. as these are external pieces of code that technically speaking not part of the core of EG and are supposed to be maintained by a 3rd party. people think that because it is included in the core that they are maintained by us. Which on a side not we should change the authors to any of the ones that are maintained or written by the core team. to "The EventGhost Team" This can be changed in the future I guess. I do not even know if the list is correct to the plugins that are shipped with EG. I would have to check.